test/fixtures/agenda-instance.js

```diff
@@ -1,28 +1,30 @@
-var connStr = process.argv[2];
-var tests = process.argv.slice(3);
-
-var path = require('path'),
-    Agenda = require( path.join(__dirname, '..', '..', 'index.js' ) ),
-    addTests = require( path.join(__dirname, 'addTests.js') );
-
-var agenda = new Agenda({ db: { address: connStr } }, function(err, collection) {
-
-	tests.forEach(function(test) {
+const Agenda = require('../../index');
```

Comment:
Aight, valid point! Seems like good to me like it is now as a transition towards XO — how was it, _Never discuss code style on a pull request again!_ :-D